src/routes/accommodationRoute.js

```diff
@@ -81,16 +82,18 @@ const fUpload = fileUpload({
 });
 
 const {
-  isSupplierAccommodation, checkForImages, checkForImagesUpdate
+  isSupplierAccommodation, checkForImages, checkForImagesUpdate,
 } = accMidd;
 const { checkToken } = userMidd;
 const {
-  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation
-} = accommodationCont;
+  addAccommodation, getAccommodation, deleteAccommodation, editAccommodation,
+} = accommodationController;
 
 app.patch('/:id', fUpload, checkToken, roles.checkHost, isSupplierAccommodation, checkForImagesUpdate, valid.editAccommodation, editAccommodation);
 app.delete('/:id', checkToken, roles.checkHost, isSupplierAccommodation, deleteAccommodation);
 app.post('/', fUpload, checkToken, roles.checkHost, valid.accommodation, checkForImages, addAccommodation);
 app.get('/', checkToken, accMidd.checkViewAccommodation, getAccommodation);
+app.get('/:id/rating', bookingsController.getAverageRating);
+app.get('/:id/feedbacks', bookingsController.getAccommodationFeedbacks);
```

Comment:
I think by getting `rating`(at line 96) we should be able to get feedback on that rating too!
A user needs to know why he gets that rating(from feedbacks),
I suggest we don't need `/:id/feedbacks`.

Reply:
Getting a rating will give just the average rating of an accommodation while feedbacks will give all the feedbacks provided on a given accommodation. I think separating them is more consistent